test/parallel/test-net-connect-options-port.js

```diff
@@ -76,14 +76,14 @@ const net = require('net');
   const expectedConnections = 72;
   let serverConnected = 0;
 
-  const server = net.createServer(common.mustCall(function(socket) {
+  const server = net.createServer(common.mustCall((socket) => {
     socket.end('ok');
     if (++serverConnected === expectedConnections) {
       server.close();
     }
   }, expectedConnections));
 
-  server.listen(0, 'localhost', common.mustCall(function() {
+  server.listen(0, 'localhost', common.mustCall(() => {
     const port = this.address().port;
```

Comment:
arrow functions do not have [separate this](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions). so please use `server.address().port` in place of `this.address().port` [ I guess I have mentioned this in the assignment slip ].